Resources/Computing/BatchSystems/Condor.py

```diff
@@ -81,22 +81,23 @@ def submitJob( self, **kwargs ):
     outputDir = kwargs['OutputDir']
     executable = kwargs['Executable']
     submitOptions = kwargs['SubmitOptions']
+    pn = kwargs['Mprocess']
 
     jdlFile = tempfile.NamedTemporaryFile( dir=outputDir, suffix=".jdl" )
     jdlFile.write("""
     Executable = %s
     Universe = vanilla
-    Requirements   = OpSys == "LINUX"
+    Requirements = Target.OpSys == "LINUX"
```

Comment:
It is suggested by our condor admin that it is a standard way to describe "Requirements" with "Target".